README.md

```diff
@@ -17,18 +17,21 @@ commits for the previous tag and generate a changelog containing the commits
 since that tag.
 
 __release.sh__ uses the TOC file to determine the package name for the project.
-You can also set the CurseForge project id (`-p`) and the WoWInterface addon
-id (`-w`) by adding the following to the TOC file:
+You can also set the CurseForge project id (`-p`), the WoWInterface addon
+id (`-w`) or the Wago project id (`-a`) by adding the following to the TOC file:
 
     ## X-Curse-Project-ID: 1234
     ## X-WoWI-ID: 5678
+    ## X-Wago-ID: 91011
 
 Your CurseForge project id can be found on the addon page in the "About Project"
 side box.
 
 Your WoWInterface addon id is in the url for the addon, eg, the "5678"
 in <https://wowinterface.com/downloads/info5678-MyAddon>.
 
+Your Wago project id can be found on the developer dashboard.
+
```

Comment:
As a nitpick suggestion, I think the example project ID in the README should follow the format of project IDs the Wago site is currently generating, unless that's planned to be changed in the near future.

An example can be seen below where the Total RP 3 project has an alphanumeric project ID on the developer dashboard.

![image](https://user-images.githubusercontent.com/287102/110917003-4d548700-8311-11eb-8a4f-65c35ca92113.png)

The example in the readme on the other hand has a fully numeric project ID which, with my 9am brain in full swing, left me double-guessing whether or not the project ID shown on the dashboard was actually the correct one to be using.

Reply:
great suggestion, will change!